src/app/builder/episode/episode-picker.component.ts

```diff
@@ -39,9 +39,25 @@ export class EpisodePickerComponent implements OnChanges, OnInit {
     this.select.emit(episode);
   }
 
+  private proxiedFeedUri() {
+    const customProxyUrl= window['ENV']['FEED_PROXY_URL'];
+    const expressProxyPath = "/proxy?url=";
+
+    let proxyUri;
+
+    if(customProxyUrl != undefined){
```

Comment:
If your styleguide would prefer a ternary here, let me know and I'll replace this.